### PR TITLE
autofoldline: add option to enable automatic foldline

### PR DESCRIPTION
**Pull request by ernierasta**
Hi!
here I have automatic folding line. I had changed few function to be universal (wanted to stay DRY). As always - ready to fix anything if required. I used the same method of counting margins as you did when counting page margins when no cs_layout is defined.

### CODE REVIEW

File: countersheet.py [hunk 1]
```diff
@@ -1377,6 +1407,7 @@ def effect(self):
                 backlayer = self.create_backlayer(svg, what, 1)
 
         docwidth = self.getViewBoxWidth(svg)
+        dochight = self.getViewBoxHeight(svg)
```
Comment (lifelike) [inline countersheet.py:1410]: spelling height

File: countersheet.py [hunk 2]
```diff
@@ -1176,6 +1186,23 @@ def addregistrationmarks(self, xregistrationmarks, yregistrationmarks,
                                       docwidth - x1, y1,
                                       docwidth - x2, y2)
 
+    def find_foldingline_style(self):
+        return self.find_style("cs_foldstyle", DEFAULT_FOLDING_LINE_STYLE)
+
+    def create_foldingline(self, docwidth, dochight):
+        margin = max(self.registrationmarkslen,
```
Comment (lifelike) [inline countersheet.py:1193]: I think this calculation should also have the registration distance added after that other pull request has been merged?

File: countersheet.py (hunk 1 above)
Comment (ernierasta) [reply countersheet.py]: fixed :-)

File: countersheet.py (hunk 2 above)
Comment (ernierasta) [reply countersheet.py:1193]: Yeah, of course you are right.

**Pull request merged**